test/built-ins/RegExp/prototype/Symbol.matchAll/this-val-non-regexp.js

```diff
@@ -1,22 +0,0 @@
-// Copyright (C) 2018 Jordan Harband. All rights reserved.
```

Comment:
This test is now: [isregexp-internal-regexp-is-false.js](https://github.com/peterwmwong/test262/blob/matchall-pww/test/built-ins/RegExp/prototype/Symbol.matchAll/isregexp-internal-regexp-is-false.js)
  - Updated according to: https://github.com/tc39/proposal-string-matchall/pull/33